lib/session/session.go

```diff
@@ -110,6 +110,48 @@ type Session struct {
 	Moderated bool `json:"moderated"`
 }
 
+// FileTransferParams contain parameters for requesting a file transfer
+type FileTransferParams struct {
+	RequestID string `json:"requestId"`
+	Approved  bool   `json:"approved"`
+	// Response denotes if this is a response to a file transfer request.
+	// If true, the Approved field will be set. True = approved
+	Response bool `json:"response"`
+	// Direction is either upload or download
+	Direction string `json:"direction"`
+	// Location is location of file to download, or where to put an upload
+	Location string `json:"location"`
```

Comment:
Would `Path` be a better name?

Reply:
i was keeping it in line with the presentation on the frontend and current file transfer (not request) params. Path does seem like a better name but a change for all the other aspects that aren't in this PR might muddy it up. I'll double check, and if its too many small changed, I'll separate into another PR